### PR TITLE
Revert "Fix whitespace in RFPDF plugin"

### PR DESCRIPTION
**Pull request by marutosi**

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://marutosi-finn-revert-rfpdf.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://marutosi-finn-revert-rfpdf.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: lib/plugins/rfpdf/lib/core/image_science.rb [hunk 1]
```diff
@@ -48,18 +48,18 @@ def getimagesize(filename)
...
       out['bits'] /= out['channels'] if out['channels']
     end
-
+    
     out
   end
-
-end
+  
+end
```
Comment (floriank) [inline lib/plugins/rfpdf/lib/core/image_science.rb:65]: no new line at the end of the file.

**Comment by floriank** [comment]
before I start going through manually: I do not get the revert commit - It seems to readd  a couple of trailing whitespace which I am not a fan of.

**Comment by marutosi** [comment]
https://github.com/naitoh/rbpdf/pull/14

**Comment by NobodysNightmare** [comment]
I don't see the connection between the PR you linked and the revert you made here.

Also no explanation why you want to revert a totally legit (looking?) commit.

**Comment by myabc** [comment]
@NobodysNightmare @marutosi @floriank I'd prefer to see #1834 merged instead.

**Pull request closed without merging**